packages/discord.js/src/util/Channels.js

```diff
@@ -65,13 +66,46 @@ function createChannel(client, data, guild, { allowUnknownGuild, fromInteraction
         case ChannelType.GuildDirectory:
           channel = new (getDirectoryChannel())(guild, data, client);
           break;
+        case ChannelType.GuildForum:
+          channel = new (getForumChannel())(guild, data, client);
+          break;
       }
       if (channel && !allowUnknownGuild) guild.channels?.cache.set(channel.id, channel);
     }
   }
   return channel;
 }
 
+/**
+ * Transforms an API guild forum tag to camel-cased guild forum tag.
+ * @param {GuildForumTag} tag The tag to transform
+ * @returns {GuildForumTag}
+ */
+function transformGuildForumTag(tag) {
+  return {
+    id: tag.id,
+    name: tag.name,
+    moderated: tag.moderated,
+    emojiId: tag.emoji_id,
+    emojiName: tag.emoji_name,
```

Comment:
@discordjs/the-big-4 should we instead make this an object with id and name? `emoji.id, emoji.name`? Discord says they will do so in a future API version so we can even think ahead of time

Reply:
If they will, yes, we should.